products/sentinel_one.py

```diff
@@ -71,17 +124,18 @@ def __init__(self, profile: str, creds_file: str, account_id: Optional[list[str]
         account_names = account_name if account_name else list()
 
         # extract account/site ID from configuration if set
-        if 'account_id' in config[profile] and config[profile]['account_id'] not in account_ids:
-            account_ids.append(config[profile]['account_id'])
+        if 'account_id' in config[self.profile] and config[self.profile]['account_id'] not in account_ids:
+            account_ids.append(config[self.profile]['account_id'])
 
-        if 'site_id' in config[profile] and config[profile]['site_id'] not in site_ids:
-            site_ids.append(config[profile]['site_id'])
+        if 'site_id' in config[self.profile] and config[self.profile]['site_id'] not in site_ids:
+            site_ids.append(config[self.profile]['site_id'])
 
-        if 'account_name' in config[profile] and config[profile]['account_name'] not in account_names:
-            account_names.append(config[profile]['account_name'])
+        if 'account_name' in config[self.profile] and config[self.profile]['account_name'] not in account_names:
+            account_names.append(config[self.profile]['account_name'])
 
         # determine site IDs to query (default is all)
         self._site_ids = site_ids
+        self._account_ids = account_ids
 
```

Comment:
```suggestion
        self._account_names = account_names
```
add account_names in case this is also specified via commandline argument. One question is if these variables need to be defined in the Product Class for S1.

Reply:
`account_names` is only used to pull site_ids (per line 169) and we don't reference that info anywhere else afterwards. The reason `self._site_ids` and `self._account_ids` exists is because `_get_default_header(self)` references those